### DISCUSSION TITLE
Possible to use Unleashed Chat as endpoint?

### DISCUSSION THREAD

**Discussion by nmorton13** [danny-avila/LibreChat#2940] [Q&A]
https://unleashed.chat has an OpenAI compatible API. Would it be possible to use as an endpoint? Docs https://unleashed.chat/api/docs and https://github.com/UnleashedChat/unleashed-sdk

**Comment by danny-avila** [comment]
I don't see why not, try it out!

**Reply by nmorton13** [reply]
Works! Except for generating the title. But I can live with that. Probably something I am missing anyway. I'll play with it more.
